be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -181,7 +190,10 @@ Status SegmentIterator::init(const StorageReadOptions& opts) {
         _output_columns = *(opts.output_columns);
     }
 
-    _remaining_vconjunct_root = opts.remaining_vconjunct_root;
+    _remaining_vconjunct_ctx = opts.remaining_vconjunct_ctx;
```

Comment:
any difference between _remaining_vconjunct_ctx and _remaining_vconjunct_root?

Reply:
`_remaining_vconjunct_root` = `_remaining_vconjunct_ctx->root()`
Got `_remaining_vconjunct_root` in `SegmentIterator::init`. Compatible before